### PR TITLE
Remove outdated irc log link

### PR DESCRIPTION
**Pull request by robx**
The logs don't seem to have survived the move to libera.

### CODE REVIEW

**Comment by tomjaguarpaw** [comment]
True, but the older logs still exist and are presumably interesting. I would prefer keeping them around. I would even more strongly prefer adding a new source of logs, if anyone knows one.

**Comment by robx** [comment]
I've updated the PR to keep the link but mention that it's not kept up-to-date.

**Review by tomjaguarpaw — APPROVED** [review]

**Comment by tomjaguarpaw** [comment]
OK, that works for me. We'll need to wait for one more committee member to approve this before merging.

**Review by TikhonJelvis — APPROVED** [review]

**Comment by TikhonJelvis** [comment]
Thanks for the fix! Sorry it took a while to merge in.

**Pull request merged**
